ui/src/services/prometheus/api.js

```diff
@@ -69,7 +69,7 @@ export function queryPrometheusRange(
   end: string,
   step: number, // step (time in seconds between 2 metrics)
   query: string, // query (the actual PromQL expression)
-) {
+): function {
```

Comment:
function type is too generic. In practice here it is returning a Promise< PrometheusQueryResult >